internal/plugins/helm/v1/init.go

```diff
@@ -218,5 +234,13 @@ func addInitCustomizations(projectName string) error {
 		return fmt.Errorf("error updating kustomization.yaml files: %v", err)
 	}
 
+	const target = "      # according to the platforms which are supported by your solution. "
+	const replace = "      # according to the platforms which are supported by your solution."
+
+	err = util.ReplaceInFile(managerFile, target, replace)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Same as the ansible plugin, lets add a TODO comment here to make sure we clean this up when Kubebuilder scaffolds are fixed.

Reply:
Updated.